tests/foreman/cli/test_import.py

```diff
@@ -1338,3 +1404,112 @@ def test_reimport_enable_rh_repos_negative(self, test_data):
                 ssh_import_org[1], files['channels']
             )
         )
+
+    def test_import_content_hosts_default(self):
+        """@test: Import all content hosts from
+        the predefined dataset
+
+        @feature: Import Content-host
+
+        @assert: Profiles for all Content Hosts created
+
+        """
+        for test_data in gen_import_chost_data():
+            with self.subTest(test_data):
+                tmp_dir = self.default_dataset[0]
+                files = dict(self.default_dataset[1])
+                files = update_csv_values(files, test_data, tmp_dir)
+                # import the prerequisities and content hosts
+                imports = import_content_hosts(files, tmp_dir)
+                for entity in imports:
+                    self.assertEqual(imports[entity][0].return_code, 0)
+                # get the sat6 mapping of the imported organizations
+                imp_orgs = get_sat6_id(
+                    Import.csv_to_dataset([files['users']]),
+                    imports['organizations'][1]
+                )
+                # now to check whether all cont. hosts appeared insatellite
+                for imp_org in imp_orgs:
+                    self.assertNotEqual(
+                        ContentHost.list(
+                            {'organization-id': imp_org['sat6']}
+                        ).stdout,
```

Comment:
Please, remove `.stdout` part as #2840 is merged. And i guess you can fit it in one line then:
```python
self.assertNotEqual(
    ContentHost.list({'organization-id': imp_org['sat6']}),
    []
)
```